src/catalog/src/information_schema/columns.rs

```diff
@@ -48,16 +48,16 @@ pub(super) struct InformationSchemaColumns {
     catalog_manager: Weak<dyn CatalogManager>,
 }
 
-const TABLE_CATALOG: &str = "table_catalog";
-const TABLE_SCHEMA: &str = "table_schema";
-const TABLE_NAME: &str = "table_name";
-const COLUMN_NAME: &str = "column_name";
-const DATA_TYPE: &str = "data_type";
-const SEMANTIC_TYPE: &str = "semantic_type";
-const COLUMN_DEFAULT: &str = "column_default";
-const IS_NULLABLE: &str = "is_nullable";
+pub const TABLE_CATALOG: &str = "table_catalog";
+pub const TABLE_SCHEMA: &str = "table_schema";
+pub const TABLE_NAME: &str = "table_name";
+pub const COLUMN_NAME: &str = "column_name";
+pub const DATA_TYPE: &str = "data_type";
+pub const SEMANTIC_TYPE: &str = "semantic_type";
+pub const COLUMN_DEFAULT: &str = "column_default";
+pub const IS_NULLABLE: &str = "is_nullable";
 const COLUMN_TYPE: &str = "column_type";
```

Comment:
```suggestion
pub const COLUMN_TYPE: &str = "column_type";
```

? Maybe `pub` or not by logically necessary, not because we need to use it elsewhere. So that we can keep concepts consistent.

Reply:
I am not sure what you mean. They are declared `pub` because I want to use them in other places.

Reply:
Yeah. I mean if these fields SHOULD BE  `pub`, we can declare all of them `pub`. Otherwise, it can be a hack to make them `pub`.

Reply:
Not all the columns are used in other places, so I don't change them all `pub` too.

Reply:
Fine. I don't have a strong feeling on this. But exposing fields roughly is a common smell in my experience.